controllers/smbservice_controller.go

```diff
@@ -41,6 +41,7 @@ type SmbServiceReconciler struct {
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list
 
+// Reconcile resources.
```

Comment:
There's a requirement to have a doc comment on every exported (capitalized) function. But because these function names are chosen by the sdk rather than us, I just was lazy and gave it a very low-value comment. I can make it more useful if you think that's a good thing. Alternatively, we can tell revive to skip the rule requiring doc comments for this file.

Reply:
Ah, it's required by revive, got it.
Well, with this explanation we can keep the redundant comments as well.. :wink: